config/katello-answers.yaml

```
@@ -89,7 +88,7 @@ foreman_proxy:
   ssl_ca: /etc/foreman-proxy/ssl_ca.pem
   ssl_cert: /etc/foreman-proxy/ssl_cert.pem
   ssl_key: /etc/foreman-proxy/ssl_key.pem
-  ssl_port: '9090'
+  ssl_port: "9090"
```

Please keep this unchanged
```suggestion
  ssl_port: '9090'
```